src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -23,6 +23,12 @@
       AdsContent.PREFOOTER_RIGHT
   };
 
+  private static final String noSkinWikiName = "adtest";
+  private static final String noSkinPageName = "Skin";
+
+  private static final String gradientWikiName = "adtest-pluto";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)